### ISSUE TITLE
Doesn't work properly

### ISSUE THREAD

**Issue by ProfPh** [kesco/SlideBack-Xposed#2]
Hello,

This is a lovely app you have with a very smooth animation but for me it only works on your app and GravityBox.

So I can slide back from your app to home screen but I can't do that for any other app.

And for gravitybox, it can swipe back from the main activity to the home screen but for any other activity inside the app (Settings etc) it can't slide back.

**Comment by kesco** [comment]
Hi, @ProfPh . It seems that the slidback module take the injection in wrong fields of activity. I will try to rewrite the injection code.
Thanks for your issue.

**Comment by kesco** [comment]
Hi, @ProfPh . I rewrote the build gradle scripts to solve the problem for I compiled the xposed jar in a wrong scope. Sorry for taking you waiting for so long. You can download a new version apk at http://repo.xposed.info/module/com.kesco.xposed.slideback

**Comment by rtk19** [comment]
Thank you for the update, now almost everything works well, except from WhatsApp and chrome and some other apps can you fix that?

**Comment by kesco** [comment]
Hi, @rtk19 . I guess the apps like WhatsApp and Chrome using difference method of inflating view so that the xposed injection can not hook the acitivity liferecycle. I will improve the hook method to solve it. 
Thanks for your issue.

**Comment by rtk19** [comment]
After a reboot it was fixed, thanks for the answer. 
But it's there a chance that you'll add customisation? Like sliding back from the right side of the screen instead of left?

**Comment by kesco** [comment]
@rtk19 , of course, I still plan to add customisation and more functions into the plugin. However, [Slideback](https://github.com/kesco/SlideBack), the swipe libary of the plugin depens on is too old and not stable. I am going to  update the library version and improve hook method in next version. And then, I will try to add more function in 0.14 version.

**Comment by rtk19** [comment]
Ok thank you for your job!

**Comment by ProfPh** [comment]
Thank you very much for your update! It's working well on most apps now.
